scopes/lanes/ui/compare/lane-compare/lane-compare.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { HTMLAttributes, useState, useCallback, useMemo } from 'react';
+import { ComponentID } from '@teambit/component-id';
+import { ComponentCompare } from '@teambit/component.ui.component-compare.component-compare';
+import {
+  ComponentCompareState,
+  ComponentCompareStateKey,
+} from '@teambit/component.ui.component-compare.models.component-compare-state';
+import { extractLazyLoadedData, MaybeLazyLoaded } from '@teambit/component.ui.component-compare.utils.lazy-loading';
+import { TabItem } from '@teambit/component.ui.component-compare.models.component-compare-props';
+import { ComponentCompareHooks } from '@teambit/component.ui.component-compare.models.component-compare-hooks';
+import { sortTabs } from '@teambit/component.ui.component-compare.utils.sort-tabs';
+import { LaneModel } from '@teambit/lanes.ui.models.lanes-model';
+import { LaneCompareState, computeStateKey } from '@teambit/lanes.ui.compare.lane-compare-state';
+import { useUpdatedUrlFromQuery } from '@teambit/component.ui.component-compare.hooks.use-component-compare-url';
+import { UseComponentType } from '@teambit/component';
+import { DrawerUI } from '@teambit/ui-foundation.ui.tree.drawer';
+import AnimateHeight from 'react-animate-height';
+import styles from './lane-compare.module.scss';
+
+export type LaneCompareProps = {
+  base: LaneModel;
+  compare: LaneModel;
+  host: string;
+  tabs?: MaybeLazyLoaded<TabItem[]>;
+  customUseComponent?: UseComponentType;
+} & HTMLAttributes<HTMLDivElement>;
+
+export function LaneCompare({ host, compare, base, tabs, customUseComponent, ...rest }: LaneCompareProps) {
+  const baseMap = useMemo(
+    () => new Map<string, ComponentID>(base.components.map((c) => [c.toStringWithoutVersion(), c])),
+    [base.components]
+  );
+  const compareMap = useMemo(
+    () => new Map<string, ComponentID>(compare.components.map((c) => [c.toStringWithoutVersion(), c])),
+    [compare.components]
+  );
+
+  const uniqueCompare = useMemo(
```

Comment:
maybe rename or comment that these are the new components that dont have a base to compare to